src/common.rs

```diff
@@ -250,7 +255,7 @@ impl XWinOptions {
         Ok(())
     }
 
-    fn setup_msvc_crt(&self, cache_dir: PathBuf) -> Result<()> {
+    fn setup_msvc_crt(&self, cache_dir: PathBuf, include_debug_libs: bool) -> Result<()> {
```

Comment:
No need to pass it around since you can access it via `self.xwin_include_debug_libs` in this method?

Reply:
Fixed